bootstrap-obvious-ci-and-miniconda.py

```diff
@@ -1,103 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Just don't remove it. We need a `master` version in many projects. Do not be obtuse here. There are many outside `conda-forge` who use that.